Jenkinsfile.nightly

```diff
@@ -0,0 +1,92 @@
+pipeline {
+    agent { label 'go'}
+    options {
+        buildDiscarder logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '', daysToKeepStr: '', numToKeepStr: '10')
+        timeout(time: 90, unit: 'MINUTES')
+    }
+    environment {
+        WORKING_DIR = "/home/jenkins/go/src/github.com/kiegroup/kogito-cloud-operator/"
+        GOPATH = "/home/jenkins/go"
+        GOCACHE = "/home/jenkins/go/.cache/go-build"
+    }
+    stages {
+        stage('Clean Workspace') {
+            steps {
+                dir ("${WORKING_DIR}") {
+                    deleteDir()
+                }
+            }
+        }
+        stage('Initialize') {
+            steps {
+                sh "mkdir -p ${WORKING_DIR} && cd ${WORKSPACE} && cp -Rap * ${WORKING_DIR}"
+                sh "set +x && oc login --token=\$(oc whoami -t) --server=${OPENSHIFT_API} --insecure-skip-tls-verify"
+            }
+        }
+        stage('Build Kogito Operator') {
+            steps {
+                dir ("${WORKING_DIR}") {
+                    sh """
+                        export GOROOT=`go env GOROOT`
+                        GO111MODULE=on 
+                        go get -u golang.org/x/lint/golint
+                        touch /etc/sub{u,g}id
+                        usermod --add-subuids 10000-75535 \$(whoami)
+                        usermod --add-subgids 10000-75535 \$(whoami)
+                        cat /etc/subuid
+                        cat /etc/subgid
+                        make image_builder=podman
+                    """
+                }
+            }
+            
+        }
+        stage('Build Kogito CLI') {
+            steps {
+                dir ("${WORKING_DIR}") {
+                    sh "make build-cli"
+                }
+            }
+        }
+        stage('Push Operator Image to Openshift Registry') {
+            steps {
+                dir ("${WORKING_DIR}") {
+                    sh """
+                        set +x && podman login -u jenkins -p \$(oc whoami -t) --tls-verify=false ${OPENSHIFT_REGISTRY}
+                        cd version/ && TAG_OPERATOR=\$(grep -m 1 'Version =' version.go) && TAG_OPERATOR=\$(echo \${TAG_OPERATOR#*=} | tr -d '"')
```

Comment:
kogito-cli is getting built, why not use kogito --version?

Reply:
Because @ricardozanini  told me that the most reliable place to get the version is in this file.

Reply:
`kogito --version` does the same thing, indeed it's a better idea. :)

Reply:
But just make sure to have the same binary installed in the machine. Or build it before running the pipeline, otherwise you will get a completely wrong version.

Reply:
Kogito CLI is using a different file from Kogito Operator for the version
- cmd/kogito/version/version.go for CLI
- version/version.go for Operator

I would stick to this version as we are sure of the operator version ...

Reply:
They are always in sync, but could have some scenarios where the CLI would have a different micro version.

Reply:
@ricardozanini So it's better to keep it that way, right?